tests/generate_test_params.py

```diff
@@ -0,0 +1,76 @@
+# This file generates (1) test cases that populate test_params.mligo,
+# and (2) the expected values from these test cases
+
+import random 
+
+# (1) Generate the random data using a fixed seed for reproducibility
+#     This random data comes in the form:
+#      - (x, y, dx, target, rounds, const_fee) *OR* 
+#      - (y, x, dy, target, rounds, const_fee)
+#     depending on if we're testing 
+#      - trade_dtez_for_dcash or 
+#      - trade_dcash_for_dtez
+
+# The data in test_params is generated with n = 50
+def generate_random_data(n):
+    # Fix the seed
+    random.seed(0)
+    accum = []
+    # Generate the following vector:
+    # (x, y, dx, target, rounds, const_fee)
+    for i in range(n):
+        x = random.randint(0,100000000000000) # between 0 and 100_000_000 XTZ
```

Comment:
10**8

Do you mutez though?

Reply:
Yes, this is something between 0 and `100_000_000_000_000 mutez`, or `100_000_000 tez` (did I understand your question right?)